src/app/pages/main/main-page/main-page.component.ts

```diff
@@ -1,9 +1,20 @@
-import { Component } from '@angular/core';
+import { TokenStorageService } from './../../../services/token-storage.service';
+import { Component, OnInit } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/SimpleSurveyProject/SimpleSurvey-Frontend/pullRequest?prid=7440283)
```suggestion
import { Component, OnInit } from "@angular/core";
```